### PR TITLE
Blog: May heartbeat

### PR DESCRIPTION
**Pull request by jendefig**
Publish today? 😬. If not, tomorrow, but one item will need to be dropped.
- [x] Header Image correct
- [x] There are no grammatical errors, jargon, or obvious stylistic issues. (For reviewer)
- [x] Have images been run through tiny.png?
- [x] Are all lines 80 characters or less?
- [x] Are Oxford commas observed?
- [x] Content Creator has done a complete review for typos/language after taking a break from content
- [x] links checked in piece
- [x] Have reviewed what the piece looks like on your own machine using yarn develop
- [x] Needs a Twitter link

### CODE REVIEW

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [inline content/blog/2022-05-16-may-22-heartbeat.md:6]:
```suggestion
  guides and frameworks on the iterative nature of AI, tons of company news,
```

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [inline content/blog/2022-05-16-may-22-heartbeat.md:7]:
```suggestion
  Dmitry on TFIR, beyond machine learning use cases and more! Welcome to May!
```

Comment (flippedcoder) [reply content/blog/2022-05-16-may-22-heartbeat.md:7]: Spelling out machine learning helps with SEO.

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [inline content/blog/2022-05-16-may-22-heartbeat.md:56]:
```suggestion
present an iterative-lifecycle framework," that is adapted to AI-centered
```

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [inline content/blog/2022-05-16-may-22-heartbeat.md:64]:
```suggestion
In addition, she discusses issues with quality assurance in deployment and the
```

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [inline content/blog/2022-05-16-may-22-heartbeat.md:127]:
```suggestion
  auto-respawning of low-cost spot/pre-emptible instances
```

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [inline content/blog/2022-05-16-may-22-heartbeat.md:168]:
```suggestion
[**Milecia McGregor**](https://twitter.com/FlippedCoding), and
```

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (jendefig) [reply content/blog/2022-05-16-may-22-heartbeat.md:127]: There's apparently not a hyphen in that word.  https://www.google.com/search?q=preemtible&oq=preemtible&aqs=chrome.0.69i59j0i10l9.5603j0j7&sourceid=chrome&ie=UTF-8

Weird to me too!

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (jendefig) [reply content/blog/2022-05-16-may-22-heartbeat.md:168]: Thanks. I keep doing this! 🙈 I did this more than once I think.  I go through the find.

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [inline content/blog/2022-05-16-may-22-heartbeat.md:249]:
```suggestion
reinventing recruitment in the deep-tech/ML field. We look forward to him helping
```

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [inline content/blog/2022-05-16-may-22-heartbeat.md:259]: This image isn't rendering.

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [inline content/blog/2022-05-16-may-22-heartbeat.md:265]:
```suggestion
## Yet another tool comparison, imagine that!
```

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [inline content/blog/2022-05-16-may-22-heartbeat.md]: Link to the Twitter thread?

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [inline content/blog/2022-05-16-may-22-heartbeat.md:294]:
```suggestion
pointing out how Jupyter Notebooks, while rightfully loved in data science work,
```

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (jendefig) [reply content/blog/2022-05-16-may-22-heartbeat.md:259]: Interesting. I've used the same code for the past three Heartbeat.  I just moved a copy of the image over. 👍

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (jendefig) [reply content/blog/2022-05-16-may-22-heartbeat.md]: I just added it.

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (flippedcoder) [reply content/blog/2022-05-16-may-22-heartbeat.md:127]: Huh! English is weird. 😅

**Comment by jendefig** [comment]
@julieg18 @rogermparent Does this new gatsby thing check the links, or is it turned off and I need to go through them one by one?

**Comment by rogermparent** [comment]
Unfortunately, our link checker is tied to the Heroku deploys and hasn't been updated to work with Gatsby Cloud since we had thought the Heroku issue was going to be more temporary than it has turned out to be. I'll locally run a check on this PR's diff so you don't have to do so manually.

**Comment by rogermparent** [comment]
Sorry for the delay! Here's the check.

```
yarn link-check-diff -f -r localhost:3000
yarn run v1.22.18
$ repo-link-check -c config/link-check/config.yml -d master -f -r localhost:3000
```
* content/blog/2022-05-16-may-22-heartbeat.md
  - https://dvc.org/doc/start/visualization (404)
  - https://www.udemy.com/course/dvc-and-git-for-data-science/ (403)

This can also be done locally on your end with `yarn link-check-diff -f` (`-f` to only show failing links), but I took it on myself just to ensure
1. this hacky library still works like this
2. the results are interpreted in a useful way as quickly as possible

I'll make inline comments on what changes are needed

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (rogermparent) [inline content/blog/2022-05-16-may-22-heartbeat.md:203]: This page is under an `experiments` subdirectory, i.e. https://dvc.org/doc/start/experiments/visualization

Also, the link would ideally be root-relative so that the advanced local link handling can be used, i.e. `/doc/start/experiments/visualization`

File: content/blog/2022-05-16-may-22-heartbeat.md
Comment (rogermparent) [inline content/blog/2022-05-16-may-22-heartbeat.md:360]: This link seems fine, Udemy is just refusing the link checker because it's either blocking bots or requiring logins.

Comment (jendefig) [reply content/blog/2022-05-16-may-22-heartbeat.md:360]: Thanks @rogermparent!!! 🙏

**Comment by jendefig** [comment]
Can I get approval on this so it can be merged? It's ready. @shcheklein @flippedcoder @rogermparent

**Review by rogermparent — APPROVED** [review]
Just reviewing to unblock, trusting that it's good to go

**Pull request merged**
